otherlibs/systhreads/thread.mli

```diff
@@ -41,6 +42,17 @@ val id : t -> int
    is an integer that identifies uniquely the thread.
    It can be used to build data structures indexed by threads. *)
 
+exception Exit
+(** Exception that can be raised by user code to initiate termination
+    of the current thread.
+    Compared to calling [{!Thread.exit} ()], raising the {!Thread.Exit}
```

Comment:
The syntax [{!Thread.exit} ()] is unfortunately not supported by ocamldoc and leaks the `{!Thread.exit}` to the final render. I propose either` {!Thread.exit}[ ()]` (to keep the link) or `[Thread.exit ()]`.

Reply:
Damn!  I rephrased: "compared to calling the {!Thread.exit} function, ...".  Gotta check what I wrote in the 5.00 version of this PR.